packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -129,6 +133,75 @@ export function isConformant(
     expect(component.find(props).length).toBeGreaterThan(1);
   });
 
+  if (skipAsPropTests !== 'all') {
+    describe('"as" prop (common)', () => {
+      test('renders the component as HTML tags or passes "as" to the next component', () => {
+        // silence element nesting warnings
+        consoleUtil.disableOnce();
+
+        const tags = ['a', 'em', 'div', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'i', 'p', 'span', 'strong'];
+
+        tags.forEach(tag => {
+          const wrapper = mount(<Component {...requiredProps} as={tag} />);
+          const component = getComponent(wrapper);
+          try {
+            expect(component.is(tag)).toEqual(true);
+          } catch (err) {
+            expect(component.type()).not.toEqual(Component);
+            expect(component.prop('as')).toEqual(tag);
+          }
+        });
+      });
+
+      if (skipAsPropTests !== 'as-component') {
```

Comment:
Noticed while trying to figure out build failures that #20946 disabled the "renders as function component" and "renders as class component" tests for all of northstar. It seems like it's only actually necessary to disable them for a subset of components (based on what failed in the [prior build](https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=237534&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55&t=cc94c182-d6f4-5dae-734d-3676c12cee2a)), so I added a way to handle that and updated the relevant components' test files.